book/src/road-map.md

```diff
@@ -0,0 +1,33 @@
+### Vendor Management
+
+- [ ] Manual SBOM upload screen (under development)
+- [ ] Role-based permissions related to SBOM upload screen (under development)
+- [ ] Vendor SBOM staleness/freshness policy tracking and enforcement (research & development)
+- [ ] Machine to machine vendor upload mechanism (research)
+
+### SBOM generation & synchronization
+- [X] Snyk
+- [X] GitHub Action (POC)
+- [ ] GitHub (in progress)
+- [ ] GitLab
+- [ ] Bitbucket
+- [ ] Syft
+- [ ] Ion Channel
+
+### SBOM storage
+- [X] S3
+- [X] File system
+
+### Enrichment
+- [X] Snyk
+- [X] EPSS
+- [ ] Ion Channel
+- [ ] OSV
+- [ ] Dependency Track
+
+### Technical Improvements
+
+- [ ] SPDX support
+- [ ] Pipeline Metrics (e.g. OTEL, Prometheus, Grafana, )
+- [ ] Cargo features
```

Comment:
What does "Cargo Features" give us?

Reply:
The ability to opt in/out of different functionality. This is helpful in reducing binary size among other things. So if your org doesn't use Snyk for example, you don't compile that in and A) your binary is smaller B) your build time is faster C) your attack surface is smaller